CHANGELOG.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 
 master
 ------
-
+- (`#23 <https://github.com/openscm/openscm-units/pull/23>`_) Add AR5 GWPs with climate-carbon feedbacks (closes `#22 <https://github.com/openscm/openscm-units/issues/22>`_)
```

Comment:
```suggestion
- (`#23 <https://github.com/openscm/openscm-units/pull/23>`_) Add AR5 GWPs with climate-carbon cycle feedbacks (closes `#22 <https://github.com/openscm/openscm-units/issues/22>`_)
```

Reply:
The name in the IPCC report is actually "... with climate-carbon feedbacks" without "cycle" (which I find a bit weird) but I just used the "official" name.